test/functionalTest/cases/0000_log_operation/log_rest.test

```diff
@@ -47,7 +47,7 @@ echo
 
 
 echo "=== 4. POST trace level"
-orionCurl --url "/log/traceLevel/101-102,105" -X "POST" --include 2> /dev/null
+orionCurl --url "/log/traceLevel/101-102,105" -X "POST" 2> /dev/null
```

Comment:
I had problems with this file when I introduced testJson.py checker... not sure of the cause, but removing the `--include` curl parameter (as in other orionCurl statement in this same .test) solved it.

Reply:
NTC